packages/gcalculator.rb

```diff
@@ -5,22 +5,26 @@ class Gcalculator < Package
   homepage 'https://wiki.gnome.org/Apps/Calculator'
   compatibility 'all'
   @_app = 'gnome-calculator'
-  @_fullver = '3.38.2'
-  @_mainver = @_fullver.rpartition('.')[0]
-  @_url = "https://download.gnome.org/sources/#{@_app}/#{@_mainver}/#{@_app}-#{@_fullver}"
-  version @_fullver
-  source_url "#{@_url}.tar.xz"
-  source_sha256 `curl -Ls #{@_url}.sha256sum | tail -n1 | cut -d ' ' -f1`.chomp
+  @_ver = '3.38.2'
+  @_url = "https://download.gnome.org/sources/#{@_app}/#{@_ver.rpartition('.')[0]}/#{@_app}-#{@_ver}"
+  version @_ver + '-1'
+  source_url @_url + '.tar.xz'
 
   depends_on 'setuptools' => :build
   depends_on 'gtk3'
   depends_on 'gtksourceview'
   depends_on 'itstool'
   depends_on 'libgee'
   depends_on 'libsoup'
+  
+  def self.preflight
+    source_sha256 `curl -Ls #{@_url}.sha256sum | tail -n1 | cut -d' ' -f1`.chomp
```

Comment:
Why is this here?

Reply:
To get the checksum only at download (not search and update)

Reply:
I thought https://github.com/skycocker/chromebrew/pull/5385 already handles this?